lib/dataset/utils/parsers.js

```diff
@@ -106,7 +106,7 @@ function parseDoubleGroupedMetric(){
     var dataset = new Dataset()
       .type('double-grouped-metric');
     each(res.result, function(record, i){
-      dataset.set([ 'Result', record[options[0][0]] + ' ' + record[options[0][1]] ], record.result);
+      dataset.set([ 'Result', String(record[options[0][0]] + ' ' + record[options[0][1]]) ], record.result);
```

Comment:
Is this redundant? `record[options[0][0]] + ' ' + record[options[0][1]]` will return a `string` no matter what.